_inc/lib/class.media-extractor.php

```diff
@@ -59,6 +59,9 @@ public static function extract( $blog_id, $post_id, $what_to_extract = self::ALL
 
 		$post = get_post( $post_id );
 		if ( ! $post instanceof WP_Post ) {
+			if ( function_exists( 'restore_current_blog' ) ) {
```

Comment:
`restore_current_blog()` has been available since WP 3.0.0 - why might it not be defined in this context?

Reply:
This is a good point for future changes, thanks @GaryJones ! This PR was meant to bring the code back in sync, but we'll definitely look at removing redundant checks in the future.